cluster-density.go

```diff
@@ -38,15 +36,6 @@ func NewClusterDensity(wh *workloads.WorkloadHelper, variant string) *cobra.Comm
 		Use:   variant,
 		Short: fmt.Sprintf("Runs %v workload", variant),
 		PreRun: func(cmd *cobra.Command, args []string) {
-			kubeClientProvider := config.NewKubeClientProvider("", "")
```

Comment:
Don't we want cluster health check in cluster-density?

Reply:
It can be enabled in all workloads https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/ocp.go#L85-L87
I removed it from cluster-density to standarize